tests/api/test_account.py

```diff
@@ -414,3 +415,148 @@ def test_set_password(user_api_client, customer_user):
 
     customer_user.refresh_from_db()
     assert customer_user.check_password(password)
+
+
+@patch('saleor.account.emails.send_password_reset_email.delay')
+def test_create_user_password_reset_email(
+        send_password_reset_mock, admin_api_client):
+    query = """
+    mutation CreateCustomer($email: String, $note: String, $send_mail: Boolean) {
+        customerCreate(input: {email: $email, note: $note, sendPasswordEmail: $send_mail}) {
+            errors {
+                field
+                message
+            }
+            user {
+                id
+                email
+                isStaff
+                isActive
+                note
+            }
+        }
+    }
+    """
+    email = 'api_user@example.com'
+    note = 'Test user'
+
+    variables = json.dumps({'email': email, 'note': note, 'send_mail': True})
+    response = admin_api_client.post(
+        reverse('api'), {'query': query, 'variables': variables})
+    content = get_graphql_content(response)
+    assert 'errors' not in content
+    data = content['data']['customerCreate']
+    assert data['errors'] == []
+    assert data['user']['email'] == email
+    assert data['user']['note'] == note
+    assert data['user']['isStaff'] is False
+    assert data['user']['isActive'] is True
+    assert send_password_reset_mock.call_count == 1
+    args, kwargs = send_password_reset_mock.call_args
+    call_context = args[0]
+    call_email = args[1]
+    assert call_email == email
+    assert 'token' in call_context
+
+
+@patch('saleor.account.emails.send_password_reset_email.delay')
+def test_create_staff_password_reset_email(
```

Comment:
Same here, I would do the necessary changes in `test_staff_create` to include the email testing there.